src/framework/theme/component/modal/modal.spec.tsx.snap

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do we really need snapshot tests for this component?

Reply:
Just for complex testing. Let it be.